orttraining/orttraining/test/training_api/common/synthetic_data_loader.cc

```diff
@@ -38,32 +40,54 @@ void RandomInts(std::vector<IntType>& rets, IntType low, IntType high) {
 
 }  // namespace
 
+template <typename T>
+void SyntheticSampleBatch::AddIntInput(const std::vector<int64_t>& shape, T low, T high) {
+  data_vector_.emplace_back(SyntheticInput(shape));
+
+  std::vector<T> values(data_vector_.back().NumOfElements());
+  RandomInts(values, low, high);
+
+  SyntheticDataVector& data = data_vector_.back().GetData();
+  data = values;
+  input_count_++;
```

Comment:
>input_count_ [](http://example.com/codeflow?start=2&length=12)

Why separate count is necessary?

Reply:
my bad, I misunderstood the meaning here. Corrected it.